src/dmd/backend/cg87.c

```diff
@@ -9,7 +9,7 @@
  * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/cg87.c, backend/cg87.c)
  */
 
-#if !SPP
+#if (SCPP && !HTOD) || MARS
```

Comment:
I don't know what this is, but I trust @WalterBright does.

Reply:
It was the result of some DMC work :-)